crates/typed-store/src/rocks/mod.rs

```diff
@@ -983,6 +984,17 @@ impl<K, V> DBMap<K, V> {
                 Self::get_int_property(rocksdb, &cf, ROCKSDB_PROPERTY_TOTAL_BLOB_FILES_SIZE)
                     .unwrap_or(METRICS_ERROR),
             );
+        let total_num_files: i64 = (0..=6)
+            .map(|level| {
```

Comment:
Could we also query the num levels in the db or is this the max value?

Reply:
This is the default value, and yes one can change it.

I didn't find a way to query db option in rust, but there is a c++ API that can do this.

Reply:
Update comment about if we every change number of layers, we need to update here as well.

Worse case, it'll just miss lower level file count.